app/views/users/show.html.erb

```diff
@@ -1,51 +1,69 @@
-      <div class="spacer10"></div>
-      <div class="row center">
-        <div class="col-sm-4"><b>82</b> total questions</div>
-        <div class="col-sm-4"><b>6</b> contributors</div>
-        <div class="col-sm-4"><b>3</b> unanswered</div>
+    <div class="col-sm-9">
+      <!-- <h3>My enrolled classes</h3>
+      <div class="box">
+        <div class="class-bg valign">
+          <h3 class="center class-title ">Stat 110: Intro to Probability</h3>
+        </div>
+        <div class="spacer10"></div>
+        <div class="row center">
+          <div class="col-sm-4"><b>82</b> total questions</div>
+          <div class="col-sm-4"><b>6</b> contributors</div>
+          <div class="col-sm-4"><b>3</b> unanswered</div>
+        </div>
       </div>
-    </div>
-    <div class="box">
-      <h3 class="center">SLS 20</h3>
-      <div class="spacer10"></div>
-      <div class="row center">
-        <div class="col-sm-4"><b>82</b> total questions</div>
-        <div class="col-sm-4"><b>6</b> contributors</div>
-        <div class="col-sm-4"><b>3</b> unanswered</div>
+      <div class="box">
+        <div class="class-bg valign">
+          <h3 class="center class-title ">SLS 20: Intro to Psych</h3>
+        </div>
+        <div class="spacer10"></div>
+        <div class="row center">
+          <div class="col-sm-4"><b>82</b> total questions</div>
+          <div class="col-sm-4"><b>6</b> contributors</div>
+          <div class="col-sm-4"><b>3</b> unanswered</div>
+        </div>
       </div>
-    </div>
-    <div class="box">
-      <h3 class="center">USW 12</h3>
-      <div class="spacer10"></div>
-      <div class="row center">
-        <div class="col-sm-4"><b>82</b> total questions</div>
-        <div class="col-sm-4"><b>6</b> contributors</div>
-        <div class="col-sm-4"><b>3</b> unanswered</div>
+      <div class="box">
+        <div class="class-bg valign">
+          <h3 class="center class-title ">USW 12: American Encounters</h3>
+        </div>
+        <div class="spacer10"></div>
+        <div class="row center">
+          <div class="col-sm-4"><b>82</b> total questions</div>
+          <div class="col-sm-4"><b>6</b> contributors</div>
+          <div class="col-sm-4"><b>3</b> unanswered</div>
+        </div>
       </div>
-    </div>
-    <br>
-    <h3>My recent questions</h3>
-    <div class="box">
-      <h4 class="center">You haven't generated any questions yet!</h4>
-      <div class="center"><%= link_to "Create one now", new_question_path, class: "btn btn-theme btn-lg" %></div>
+      <br> -->
+      <h3>Your recent questions</h3>
+      <% if current_user.questions.count < 1 %>
+      <div class="box">
+        <h4 class="center">You haven't generated any questions yet!</h4>
+        <div class="center"><%= link_to "Create one now", new_question_path, class: "btn btn-theme btn-lg" %></div>
+      </div>
+      <% else %>
+      <p>Plug in user questions here</p>
+
+      <% end %>
     </div>
   </div>
-</div>
+</div>
```

Comment:
This symbol is a reminder that the file should end with a newline character; there are a few of those in this package of commits.